dashboard/test/models/pd/application/teacher_application_test.rb

```diff
@@ -869,7 +869,7 @@ class TeacherApplicationTest < ActiveSupport::TestCase
       assert_equal(YES, application.response_scores_hash[:meets_scholarship_criteria_scores][:underrepresented_minority_percent])
     end
 
-    test 'scholarship criteria for rp guardrail percents rp did not specify guardrails' do
+    test 'scholarship criteria uses default guardrails when regaionl partner does not specify' do
```

Comment:
nit: `regional`

Reply:
Ack, that's embarrassing! Thanks for catching that!

Reply:
I've done things much more embarrassing, that then get merged. So no worries